pkg/cache/v3/linear.go

```diff
@@ -241,82 +267,114 @@ func (cache *LinearCache) computeResourceChange(sub Subscription, ignoreReturned
 	return changedResources, removedResources, nil
 }
 
+func computeSotwStableVersion(versionMap map[string]string) string {
+	// To enforce a stable hash we need to have an ordered vision of the map.
+	// TODO(valerian-roche): review if using https://pkg.go.dev/github.com/mitchellh/hashstructure/v2 is possible to remove the need to sort.
```

Comment:
what would motivate you to execute on this todo? fwiw I think the current code based on fnv is fine. Sorting is going to be cheap here, and unless you end up seeing it in profiles, I'd just throw this todo away.

Reply:
Cleaned-up for now. I'll see in profiles if this sort ends up being impactful